murano/tutorials/user-service.md

```diff
@@ -1,32 +1,32 @@
 ---
-# Sphinx - User Management
+# User Management
 ---
 
 * [Overview](#head_overview)
 * [Tutorial example in scripting system](#head_tutorial_example)
 * [Reference for example](#head_reference)
 
 ## <span id="head_overview">Overview</span>
-This system is to manage users under Murano solution. It supports user authentication, role-based access control and storage per user.
+This system is to manage users under the Murano solution. It supports user authentication, role-based access control, and storage per user.
 
 #### User Authentication
-* basic (email & password)
-* token based
-* social auth
+* Basic (email & password)
+* Token based
+* Social auth
 
 #### User Permission
 We support user permission based on [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control).
-In our system, there are three concrete elements &rsquo;role&rsquo;, &rsquo;user&rsquo; and &rsquo;endpoint&rsquo;. According to RBAC, we can control user&rsquo;s access to endpoint by concept below:
+In our system, there are three concrete elements: &rsquo;role&rsquo;, &rsquo;user&rsquo;, and &rsquo;endpoint&rsquo;. According to RBAC, we can control a user&rsquo;s access to endpoint with the concept below:
 
 1. Assign endpoint to role
 2. Assign role to user
-3. User can access the endpoint his role has
+3. User can access the endpoint of his role
 
 <div align="center">
   <img src="https://github.com/unahouExosite/docs/blob/MUR-298/murano/assets/usm_rbac_1.png"><br>
 </div>
 
-However, URL can be different in variable practically. For example, &rsquo;*device/1/info*&rsquo; and &rsquo;*device/2/info*&rsquo; are different literally but can be considered the same endpoint. We don&rsquo;t want to create two endpoints for such difference, that is why we add the fourth element called &rsquo;parameter&rsquo;.
```

Comment:
Part of the endpoint URL can be dynamic.

For example, by creating an endpoint like /device/{resource id}/info.
This endpoint can serve requests like:
/device/48d705d480f1/info
/device/48d705d480f2/info

48d705d480f1, 48d705d480f2 are serial numbers of the hardware devices

Stephen Chang